ssas/service/server.go

```diff
@@ -244,12 +247,13 @@ type CommonClaims struct {
 	SystemID string `json:"sys,omitempty"`
 	// In a registration token, GroupIDs contains a list of all groups this user is authorized to manage
 	GroupIDs []string `json:"gid,omitempty"`
-	Data     string   `json:"dat,omitempty"`
+	Data     string   `json:"group_data,omitempty"`
```

Comment:
Consumers of SSAS v1 depend on the `dat` claim.

Reply:
okay thought so, the ticket seemed like it was calling out to change this field